src/data/builder-tools.js

```diff
@@ -770,6 +770,14 @@ export const Showcases = [
     getstarted: "/docs/get-started/aiken",
     tags: ["favorite", "cli", "plutus"],
   },
+  {
+    title: "Acca",
+    description: "Library extending Aiken's standard library",
+    preview: require("./builder-tools/aiken.png"),
+    website: "https://github.com/Cardano-Fans/acca",
+    getstarted: null,
+    tags: ["favorite", "plutus", "library", "aiken"],
```

Comment:
3 days old repo is tagged as fav because..?

Reply:
Because `acca` is composed of the people's favourite Aiken library functions I think 🤓🤣

Reply:
OK, I see.  I had entered a review comment pointing out the problems with the tags, but it didn't get posted I guess because the review was never submitted.  Doing that now.